docs/user-guide/global-configurations/lock-deployment-config.md

```diff
@@ -0,0 +1,103 @@
+
+The 'protect configuration' feature is meant to verify the edits by introducing an approval flow for any changes made to the configuration files, i.e., Deployment template, ConfigMaps, and Secrets. This is performed at application-level.
+
+Whereas, the 'lock deployment config' feature goes one step further. It is meant to prevent any edits to specific keys by non-super-admins. This applies only to deployment templates and is performed at global-level.
+
+---
+
+## Locking Deployment Keys
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to lock deployment keys.
+{% endhint %}
+
+1. Go to **Global Configurations** → **Lock Deployment Config**. Click **Configure Lock**.
+
+    ![Figure 2: Configure Lock Button](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/lock-deployment-config.jpg)
+
+2. (Optional) Click **Refer Values.YAML** to check which keys you wish to lock.
+
+    ![Figure 3: Values.YAML File](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/values-yaml.jpg)
+
+3. Enter the keys inside the editor on the left-hand side, e.g., `resource.limits`.
+
+    ![Figure 4: Referring Values.YAML File](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/side-by-side.jpg)
+
+    Since the template is in a YAML format, use <a href="https://goessner.net/articles/JsonPath/index.html" target="_blank">JSONpath expressions</a> to enter specific keys, lists, or objects to lock.
+
+    ![Figure 5: Referring Values.YAML File for Locking Keys](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/autoscaling-lock.jpg)
+
+4. Click **Save**. 
+
+    ![Figure 6: Saving Locked Keys](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/saving-locked-keys.jpg)
+
+5. A confirmation dialog box would appear. Read it and click **Confirm**.
+
+    ![Figure 7: Confirmation Dialog](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/confirmation.jpg)
+
+---
+
+## Result
+
+While super-admins can directly edit the locked keys, let's look at a scenario where a user (non-super-admin) tries to edit the same in an [unprotected](../../user-guide/creating-application/config-approval.md) base deployment template.
+
+* User can hide/unhide the locked keys as shown below.
+
+    ![Figure 8: Hiding Locked Keys](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/hide-locked-keys.gif)
+
+{% hint style="info" %}
+<span><img src="https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/mode.jpg" alt="GUI or YAML Mode"></span> <br />
+If you have selected 'Basic' mode instead of 'Advanced (YAML)', only unlocked keys would appear.
```

Comment:
Even if some keys from the `Basic` view are locked, all keys included in `Basic` view will keep showing. The user can change values from the UI but would not be able to save changes to any locked key.